module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,30 @@
+#include "vowel.hpp"
+#include "algorithm"
+#include <stdio.h>
+#include <string.h>
+#include <ctype.h>
```

Comment:
Firstly I think newline would be nice between header include and standard library includes.
Secondly Why  #include "algorithm" instead of #include <algorithm>?
Thirdly it's highly recommended to sort includes alphabetically.